go-controller/pkg/ovn/egressip.go

```diff
@@ -991,7 +988,7 @@ func (oc *Controller) addPodEgressIPAssignments(name string, statusAssignments [
 		}
 	}
 	for _, status := range remainingAssignments {
-		klog.V(5).Infof("Adding pod egress IP status: %v for EgressIP: %s and pod: %s/%s", status, name, pod.Name, pod.Namespace)
+		klog.Infof("Adding pod egress IP status: %v for EgressIP: %s and pod: %s/%s", status, name, pod.Name, pod.Namespace)
```

Comment:
so why is it that this useful log is not printed by default and its log 5? :D Is it because it will generate too much noise when egressIPs move around nodes or pods are deleted/created? @flavio-fernandes @kyrtapz 
I can remove this change from this PR, but for me this line was very useful to see what is being assigned to what pod, but perhaps with the latest libovsdb changes we are anyways able to see each transact happening so maybe we are covered.

Reply:
> so why is it that this useful log is not printed by default and its log 5? :D Is it because it will generate too much noise when egressIPs move around nodes or pods are deleted/created?

I think the log level was intentional as it can result in a lot of noise. It is printed every time there is a egress IP configured for a pod. With big clusters and ovn-k restarts it could potential result in a **lot** of log messages.

Reply:
yeah, I would like this log to be there, but Alex convinced me it was too chatty to be useful.